.github/workflows/test_e2eshark.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     runs-on: nodai-amdgpu-mi250-x86-64
     env:
       E2E_VENV_DIR: ${{ github.workspace }}/test-suite_venv
+      HF_TOKEN: ${{ secrets.HF_TOKEN }}
+      AZ_PRIVATE_CONNECTION: ${{ secrets.AZ_PRIVATE_CONNECTION }}
```

Comment:
Please qualify this with which storage account this private connection string is used for, e.g. `ONNXPRIVATESTORAGE_AZ_PRIVATE_CONNECTION`. I wasn't sure what needed updating in this repo after I rotated keys last time.